src/pytest_benchmark/plugin.py

```diff
@@ -1053,6 +1069,7 @@ def pytest_benchmark_generate_json(config, benchmarks, include_data):
                 "group": bench.group,
                 "name": bench.name,
                 "fullname": bench.fullname,
+                "params": bench.params,
```

Comment:
Would it make sense to also have "param" here? ... a convenient mashup of params (and otherwise hard to recreate because it depends on some ordering).